LICENSE

```diff
@@ -1,2 +1,202 @@
-Unless specifically indicated otherwise in a file, files are licensed
-under the Apache 2.0 license, as can be found in: apache-2.0.txt
+
```

Comment:
You probably know the current best practice better than me @danh-arm, but I was told previously that it was best to preserve the filename of the original license file, and use LICENSE to point to it. This change reverts that - but I guess you know the most recent guidelines/guidance says the contrary?

Reply:
I'm following the [Arm Open Source Office guidance](https://confluence.arm.com/display/OSS/Licenses+and+Notices). I can't see any a good reason for preserving the filename.

Reply:
Well the reasoning before was to preserve the licence byte for byte, including the filename, so that's what we did.

I think that predated any formal policy or the existence of the OSO.